app/views/shared/_feed.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if @feed_items.any? %>
+  <ol class="microposts">
+    <%= render partial: 'shared/feed_item', collection: @feed_items %>
```

Comment:
A lot of places use single quote in views file. You can ignore them this task but next task, let's use double quotes :ok_hand: